core/src/main/scala/io/chrisdavenport/epimetheus/Counter.scala

```diff
@@ -30,140 +30,148 @@ import scala.annotation.tailrec
   * An Example of a Counter with Labels:
   * {{{
   *   for {
-  *     cr <- CollectorRegistry.build[IO]
-  *     counter <- Counter.labelled(cr, "example_total", "Example Counter", Sized("foo"), {s: String => Sized(s)})
+  *     pr <- PrometheusRegistry.build[IO]
+  *     counter <- Counter.labelled(pr, "example_total", "Example Counter", Sized("foo"), {s: String => Sized(s)})
   *     _ <- counter.label("bar").inc
   *     _ <- counter.label("baz").inc
   *   } yield ()
   * }}}
   */
-sealed abstract class Counter[F[_]]{
+sealed abstract class Counter[F[_]] {
 
-  /**
-   * Access to the current value of this [[Counter]].
-   */
-  def get: F[Double]
-
-  /**
-   * Increment the value of this [[Counter]] by 1.
-   */
+  /** Increment the value of this [[Counter]] by 1.
+    */
   def inc: F[Unit]
 
-  /**
-   * Increment the value of this counter by the provided value.
-   *
-   * @param d The value to increase the [[Counter]] by.
-   *
-   */
+  /** Increment the value of this counter by the provided value.
+    *
+    * @param d The value to increase the [[Counter]] by.
+    */
   def incBy(d: Double): F[Unit]
 
-  def mapK[G[_]](fk: F ~> G): Counter[G] = new Counter.MapKCounter[F, G](this, fk)
+  def mapK[G[_]](fk: F ~> G): Counter[G] =
+    new Counter.MapKCounter[F, G](this, fk)
 
   private[epimetheus] def asJava: F[JCounter]
 }
 
-/**
- * Counter Constructors, and Unsafe Counter Access
- */
+/** Counter Constructors, and Unsafe Counter Access
+  */
 object Counter {
 
-  /**
-   * Constructor for a Counter with no labels.
-   *
-   * @param cr CollectorRegistry this [[Counter]] will be registered with
-   * @param name The name of the Counter -  By convention, the names of Counters are suffixed by `_total`.
-   * @param help The help string of the metric
-   */
-  def noLabels[F[_]: Sync](cr: CollectorRegistry[F], name: Name, help: String): F[Counter[F]] = for {
-    c <- Sync[F].delay(JCounter.build().name(name.getName).help(help))
-    out <- Sync[F].delay(c.register(CollectorRegistry.Unsafe.asJava(cr)))
+  /** Constructor for a Counter with no labels.
+    *
+    * @param pr PrometheusRegistry this [[Counter]] will be registered with
+    * @param name The name of the Counter -  By convention, the names of Counters are suffixed by `_total`.
+    * @param help The help string of the metric
+    */
+  def noLabels[F[_]: Sync](
```

Comment:
Since we're making breaking changes anyway, is this a good opportunity to introduce a metrics construction data type (like [`Ref.Make`](https://github.com/typelevel/cats-effect/blob/623178c308600764ddcd55015de9107fb40e7a70/kernel/shared/src/main/scala/cats/effect/kernel/Ref.scala#L198) in cats-effect) so that users don't have to thread `Sync[F]` through to as many places?

Reply:
I dont really have a horse in this race, folks can make this disappear wherever. Doing so in libs, is just moving the Sync reference. I dont think it should be this PR.

Reply:
Yeah, I'm not against the `Ref.Make` idea. But I too don't think this PR is the best place to do that.

Reply:
I agree, it should be a different PR. (But hopefully before there's a release)